src/Symfony/Component/Form/Extension/Core/DataTransformer/IntegerToLocalizedStringTransformer.php

```diff
@@ -22,17 +22,21 @@ class IntegerToLocalizedStringTransformer extends NumberToLocalizedStringTransfo
     /**
      * Constructs a transformer.
      *
-     * @param int  $scale        Unused
+     * @param int  $scale        Number of decimals
```

Comment:
For an integer, scale should always be 0. If not, well, you don't get an integer. This change looks wrong to me.